backend/src/govsg/models/govsg-verification.ts

```diff
@@ -25,7 +25,7 @@ export class GovsgVerification extends Model<GovsgVerification> {
   @Column({ type: DataType.BIGINT, allowNull: false })
   govsgMessageId: number
 
-  @Column({ type: DataType.STRING, allowNull: false })
+  @Column({ type: DataType.STRING })
```

Comment:
nit: Do you think we should make the default behaviour (`allowNull: true`) explicit? I think it would improve readability

Reply:
Probably not, otherwise we'll have to add all missing `allowNull: true` to be consistent.